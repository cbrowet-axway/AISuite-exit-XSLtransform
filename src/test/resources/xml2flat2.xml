<?xml version="1.0" encoding="ISO-8859-1" standalone="yes"?>
<pivot>
  <transaction>
    <enteteAxway>
      <axway_type_cre>CRE003</axway_type_cre>
      <axway_code_enregistrement>CRE003</axway_code_enregistrement>
      <axway_code_instance></axway_code_instance>
      <axway_code_annulation></axway_code_annulation>
      <axway_code_lot>A050021716</axway_code_lot>
      <axway_code_compostage></axway_code_compostage>
      <axway_dar></axway_dar>
    </enteteAxway>
    <ref_doc_no>A050021716</ref_doc_no>
    <type_doc>FACTURE</type_doc>
    <ref_appli_source>CLEAR</ref_appli_source>
    <company>A0</company>
    <mov_dpt>A0</mov_dpt>
    <mov_sysdate>20210608</mov_sysdate>
    <mov_date_mov>20210608</mov_date_mov>
    <journal id="V2" codeTVA="V25">
      <ecriture>
        <item_num>10</item_num>
        <origine_flux>mouvement_vente</origine_flux>
        <mov_invoice>A050021716</mov_invoice>
        <mov_acct>708852</mov_acct>
        <mov_db>0</mov_db>
        <mov_cr>52351971</mov_cr>
        <mov_sense>C</mov_sense>
        <mov_cur>EUR</mov_cur>
        <mov_name>FA/A050021716/SUEZ EAU FRANCE PB</mov_name>
        <mov_initial>FACT</mov_initial>
        <mov_transac>FACT</mov_transac>
        <mov_ledger>A0500679</mov_ledger>
        <mov_a1>A06041</mov_a1>
        <mov_a2>7870</mov_a2>
        <mov_mode></mov_mode>
        <mov_code></mov_code>
        <mov_date_ech></mov_date_ech>
        <mov_date_init>20210608</mov_date_init>
        <mov_tax_code></mov_tax_code>
        <mov_tax_basis>0</mov_tax_basis>
        <mov_type>1</mov_type>
        <mov_a3></mov_a3>
        <mov_proj>9999</mov_proj>
        <mov_agent>A050SI</mov_agent>
        <mov_order>Information non c</mov_order>
        <mov_seq></mov_seq>
        <mov_led_type>CLNT</mov_led_type>
        <mov_cheque></mov_cheque>
        <mov_qty></mov_qty>
        <mov_date_1></mov_date_1>
        <mov_date_2></mov_date_2>
        <mov_extrait></mov_extrait>
        <mov_ident_doc></mov_ident_doc>
        <mov_mar_zic8></mov_mar_zic8>
        <mov_mar_zic9></mov_mar_zic9>
        <mov_mar_zic10></mov_mar_zic10>
        <mov_lot_type></mov_lot_type>
        <sous_traitance_directe></sous_traitance_directe>
        <objet_matiere></objet_matiere>
        <ref_facture_avoir></ref_facture_avoir>
        <flag_avoir_multi_facture></flag_avoir_multi_facture>
      </ecriture>
      <ecriture>
        <item_num>20</item_num>
        <origine_flux>mouvement_vente</origine_flux>
        <mov_invoice>A050021716</mov_invoice>
        <mov_acct>445725</mov_acct>
        <mov_db>0</mov_db>
        <mov_cr>10470394</mov_cr>
        <mov_sense>C</mov_sense>
        <mov_cur>EUR</mov_cur>
        <mov_name>FA/A050021716/SUEZ EAU FRANCE PB</mov_name>
        <mov_initial>FACT</mov_initial>
        <mov_transac>FACT</mov_transac>
        <mov_ledger>A0500679</mov_ledger>
        <mov_a1></mov_a1>
        <mov_a2></mov_a2>
        <mov_mode></mov_mode>
        <mov_code></mov_code>
        <mov_date_ech></mov_date_ech>
        <mov_date_init>20210608</mov_date_init>
        <mov_tax_code>V25</mov_tax_code>
        <mov_tax_basis>52351971</mov_tax_basis>
        <mov_type>2</mov_type>
        <mov_a3></mov_a3>
        <mov_proj></mov_proj>
        <mov_agent>A050SI</mov_agent>
        <mov_order>Information non c</mov_order>
        <mov_seq></mov_seq>
        <mov_led_type>CLNT</mov_led_type>
        <mov_cheque></mov_cheque>
        <mov_qty></mov_qty>
        <mov_date_1></mov_date_1>
        <mov_date_2></mov_date_2>
        <mov_extrait></mov_extrait>
        <mov_ident_doc></mov_ident_doc>
        <mov_mar_zic8></mov_mar_zic8>
        <mov_mar_zic9></mov_mar_zic9>
        <mov_mar_zic10></mov_mar_zic10>
        <mov_lot_type></mov_lot_type>
        <sous_traitance_directe></sous_traitance_directe>
        <objet_matiere></objet_matiere>
        <ref_facture_avoir></ref_facture_avoir>
        <flag_avoir_multi_facture></flag_avoir_multi_facture>
      </ecriture>
    </journal>
  </transaction>
</pivot>
