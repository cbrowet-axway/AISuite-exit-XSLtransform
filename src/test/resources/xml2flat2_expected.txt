CRE003                   CRE003                                             A050021716                                                                                                                                                                                                                      A050021716               FACTURE                  CLEAR                    mouvement_vente          10  A0  V2 A0  2021060820210608A050021716   708852  0000000000000000005235197100CEURFA/A050021716/SUEZ EAU FRANCE PBFACTFACTA0500679A06041      7870                    202106081V25  00000000000000            9999        A050SI  Information non c      CLNT          00000000000000                                                                                                                                                                                                                                                                        
CRE003                   CRE003                                             A050021716                                                                                                                                                                                                                      A050021716               FACTURE                  CLEAR                    mouvement_vente          20  A0  V2 A0  2021060820210608A050021716   445725  0000000000000000001047039400CEURFA/A050021716/SUEZ EAU FRANCE PBFACTFACTA0500679                                    202106082V25  00005235197100                        A050SI  Information non c      CLNT          00000000000000                                                                                                                                                                                                                                                                        
